From d9663b376b66c5754298b21c3592f0a49c030f88 Mon Sep 17 00:00:00 2001
From: dghart <dghart david@4Pane.co.uk>
Date: Tue, 3 Mar 2020 13:39:02 +0000
Subject: [PATCH] Compilation fix for ubuntu xenial (16.04) and similar

The ssh version check was wrong (see https://github.com/eranif/codelite/issues/2443).
Bump it from 0.6.1 to a (guestimated) 0.6.5.
---
 CodeLite/cl_ssh.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CodeLite/cl_ssh.cpp b/CodeLite/cl_ssh.cpp
index 824804e30..f5accfde9 100644
--- a/CodeLite/cl_ssh.cpp
+++ b/CodeLite/cl_ssh.cpp
@@ -106,7 +106,7 @@ bool clSSH::AuthenticateServer(wxString& message)
 
     message.Clear();
 
-#if LIBSSH_VERSION_INT < SSH_VERSION_INT(0, 6, 1)
+#if LIBSSH_VERSION_INT < SSH_VERSION_INT(0, 7, 7)
     int hlen = 0;
     hlen = ssh_get_pubkey_hash(m_session, &hash);
     if(hlen < 0) {

